// src/api/05_product.js
//"use strict";
/*jshint esversion: 6 */
const axios = require("axios").default;


// Exportieren es darf auch anonym sein, ist denk ich nicht so schlimm
module.exports.search = function search(term) {
    return axios.get("https://api.nal.usda.gov/fdc/v1/foods/search", {
            params: {
                api_key: "PtTpskZ8Xa5Mixq7qAhs9LizA9v9jSMHRzwebVTI",
                query: term
            }
        })
        .then(function(response) {
            return response.data.foods;
        });
};

// Nur die Daten daher muss man auhc Module.exports verwenden 
module.exports.infos = function info(fdcId) {
    return axios.get("https://api.nal.usda.gov/fdc/v1/food/" + fdcId, {
            params: {
                api_key: "PtTpskZ8Xa5Mixq7qAhs9LizA9v9jSMHRzwebVTI"
            }
        })
        .then(function(response) {
            return response.data.description;
            // und hier dann halt die alles was in den Daten drinnensteht vorher aber obenstehendes ausklammen
            //return response.data;
        });
}

// info(1103344).then(function(p) {
//     console.log(p);
// });
